feat(data): Hyperjump fuel cost varies by mass (#123)
endless-sky/Endless-Sky-Delta: data/human/outfits.txt

	"outfit space" -20
	"jump speed" .2
	"jump fuel" 100
	"jump mass cost" 5
	"jump base mass" 900
	"hyperdrive" 1
	description "This mysterious box sits quietly at the heart of your ship, thinking its own inscrutable thoughts. When it comes time to travel between the stars it sets off a nuclear fusion reaction powerful enough to level an entire city, and then somehow tucks the resulting explosion in between the folds of regular space-time like a wasp stinging an oak leaf to lay her eggs inside. Your ship rides the resulting ripple in the fabric of space."

> 	description "	The fuel required to power hyperdrives varies depending on the size of the ship. So long as the ship's total mass including cargo is less than 900, then each hyperjump will cost less than 100. How much less depends on how light the ship is. Ships that have more than 900 total mass will see the fuel per jump increase significantly."

Reviewer: ```suggestion
	description "	The fuel required to power hyperdrives varies depending on the size of the ship. So long as the ship's total mass including cargo is less than 900, then each hyperjump will cost less than 100 fuel. How much less depends on how light the ship is. Ships that have more than 900 total mass will see the fuel per jump increase significantly."
```

	"outfit space" -30
	"scram drive" .2
	"jump fuel" 150
	"jump mass cost" 5
	"jump base mass" 900
	"hyperdrive" 1
	description "A scram drive is a specially designed hyperdrive that works even if your ship is traveling at a much higher speed, at the expense of using more fuel. If you are being pursued by pirates (or just want to get somewhere in a hurry) that extra bit of speed can make a big difference."

> 	description "	The fuel required to power scramdrives varies depending on the size of the ship. So long as the ship's total mass including cargo is less than 900, then each hyperjump will cost less than 150. How much less depends on how light the ship is. Ships that have more than 900 total mass will see the fuel per jump increase significantly."

Reviewer: ```suggestion
	description "	The fuel required to power scram drives varies depending on the size of the ship. So long as the ship's total mass including cargo is less than 900, then each hyperjump will cost less than 150 fuel. How much less depends on how light the ship is. Ships that have more than 900 total mass will see the fuel per jump increase significantly."
```
Also, capitalization of Scram Drive?

	"atmosphere scan" 100
	description "This is Navy technology, a collection of scanners designed to fit perfectly inside one of their surveillance drones. Since the outputs are fairly standard, you could connect it to your own ship's systems if you want. Installing more than one increases the scan range and speed."

> outfit "Cockpit in-flight Mass Display"

Reviewer: Is this, and the other ones, too long?

	category "Systems"
	cost 5000
	thumbnail "outfit/tactical scanner"
	"mass" 0
	"outfit space" 0
	"Mass Display" 1

> 	description "This innocuous device is so compact as to barely count as an outfit at all. Some pilot find it useful to know what their ship's current mass is while in-flight, and this little display adds that information to their Heads Up Display."

Reviewer: ```suggestion
	description "This innocuous device is so compact as to barely count as an outfit at all. Some pilots find it useful to know what their ship's current mass is while in-flight, and this little display adds that information to their Heads Up Display."
```

outfit "Cockpit Hyperjump Fuel Display"
	category "Systems"
	cost 11000
	thumbnail "outfit/tactical scanner"
	"mass" 0
	"outfit space" 0
	"Hyperdrive Fuel Cost Display" 1

> 	description "This innocuous device is so compact as to barely count as an outfit at all. It's sole purpose is to allow pilots to customize their heads-up display to provide their flight computer's current estimate of the amount of fuel they need to make a hyperjump."

Reviewer: ```suggestion
	description "This innocuous device is so compact as to barely count as an outfit at all. Its sole purpose is to allow pilots to customize their heads-up display to provide their flight computer's current estimate of the amount of fuel they need to make a hyperjump."
```

> outfit "Scram drive Fuel Display"

Reviewer: One more thing: should Scram drive be capitalized? I think it makes sense for this (and the Jump Drive one) not to be.

Author: I am of the opinion that outfit names are proper names and/or Titles. Might be a case for them being lowercase in a sentence, but I find it just looks and sounds bad when the stand alone title in the outfitter is lowercase.

Or were you referring to the attribute? If so, yes, all the attributes are now lower case.

Reviewer: I'm wondering if this outfit name and the Jump drive one should be like this:
```suggestion
outfit "Scram Drive Fuel Display"
```
You seem to be agreeing with that in your comment?

	category "Systems"
	cost 17000
	thumbnail "outfit/tactical scanner"
	"mass" 0
	"outfit space" 0
	"Scram drive Fuel Cost Display" 1

> 	description "This innocuous device is so compact as to barely count as an outfit at all. It's sole purpose is to allow pilots to customize their heads-up display to provide their flight computer's current estimate of the amount of fuel they need to make a hyperjump with a scram drive."

Reviewer: Missed one.
```suggestion
	description "This innocuous device is so compact as to barely count as an outfit at all. Its sole purpose is to allow pilots to customize their heads-up display to provide their flight computer's current estimate of the amount of fuel they need to make a hyperjump with a scram drive."
```

outfit "Cockpit Jump drive Fuel Display"
	category "Systems"
	cost 23000
	thumbnail "outfit/tactical scanner"
	"mass" 0
	"outfit space" 0
	"Jump drive Fuel Cost Display" 1

> 	description "This innocuous device is so compact as to barely count as an outfit at all. It's sole purpose is to allow pilots to customize their heads-up display to provide their flight computer's current estimate of the amount of fuel they need to make a hyperjump with a scram drive."

Reviewer: ```suggestion
	description "This innocuous device is so compact as to barely count as an outfit at all. Its sole purpose is to allow pilots to customize their heads-up display to provide their flight computer's current estimate of the amount of fuel they need to make a hyperjump with a scram drive."
```

> 	description "This innocuous device is so compact as to barely count as an outfit at all. Its sole purpose is to allow pilots to customize their heads-up display to provide their flight computer's current estimate of the amount of fuel they need to make a hyperjump with a scram drive."

Reviewer: Oops.
```suggestion
	description "This innocuous device is so compact as to barely count as an outfit at all. Its sole purpose is to allow pilots to customize their heads-up display to provide their flight computer's current estimate of the amount of fuel they need to make a hyperjump with a jump drive."
```
